### PR TITLE
Cloudformationtemplate

### PR DESCRIPTION
**Pull request by provavi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +0,0 @@
-# api.media
```
Comment (provkartheek) [inline README.md]: Why remove the README?

File: cloudformationtemplate.yml [hunk 2]
```diff
@@ -9,7 +9,7 @@ Parameters:
   AMI:
     Description: Linux AMI
     Type: 'AWS::EC2::Image::Id'
-    Default: ami-7172b611
+    Default: ami-e6922786
```
Comment (provkartheek) [inline cloudformationtemplate.yml]: Put a TODO or comment over here that we'll be replacing this with a hardend AMI which can perform healthchecks with firewall.

File: cloudformationtemplate.yml [hunk 3]
```diff
@@ -63,108 +63,31 @@ Mappings:
...
-    CreationPolicy:
-      ResourceSignal:
-        Timeout: PT15M
     UpdatePolicy:
       AutoScalingRollingUpdate:
         MinInstancesInService: '1'
         MaxBatchSize: '1'
         PauseTime: PT15M
-        WaitOnResourceSignals: 'true'
+        WaitOnResourceSignals: 'false'
```
Comment (provkartheek) [inline cloudformationtemplate.yml]: Don't we want this to be true?
Wait for Instance to be ready before updating the ASG.

File: cloudformationtemplate.yml [hunk 4]
```diff
@@ -178,29 +101,22 @@ Resources:
...
-            - |
-              yum update -y aws-cfn-bootstrap
-            - '/opt/aws/bin/cfn-init -v '
-            - '         --stack '
-            - !Ref 'AWS::StackName'
-            - '         --resource LaunchConfig '
-            - '         --region '
-            - !Ref 'AWS::Region'
-            - |+
+      UserData:
```
Comment (provkartheek) [inline cloudformationtemplate.yml]: Comment that the "firewalld disable is temporary till a new hardened AMI comes in play".

File: pom.xml [hunk 5]
```diff
@@ -1,90 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (provkartheek) [inline pom.xml]: Don't delete this. This pom is needed for the Media Service.

File: src/main/java/org/providence/api/media/MediaServiceApplication.java [hunk 6]
```diff
@@ -1,141 +0,0 @@
-/*
```
Comment (provkartheek) [inline src/main/java/org/providence/api/media/MediaServiceApplication.java]: Alter your git check-in to leave the service as is,
This commit is basically deleting the service and adding the cloudformation script.

File: cloudformationtemplate.yml (hunk 2 above)
Comment (provavi) [inline cloudformationtemplate.yml]: TODO

File: cloudformationtemplate.yml (hunk 3 above)
Comment (provavi) [inline cloudformationtemplate.yml]: Changed it to true

**Pull request closed without merging**